### PR TITLE
Adding font

### PR DESCRIPTION
**Pull request by RaheemJnr**
i added a font

### CODE REVIEW

File: screens/src/main/java/com/rjnr/screens/ui/screen/detail_screen/DetailScreen.kt [hunk 1]
```diff
@@ -23,13 +26,30 @@ fun DetailScreen(modifier: Modifier = Modifier, screen: DetailScreen) {
 }
 
 @Composable
-private fun UI(modifier: Modifier) {
+private fun UI(
```
Comment (jcaiqueoliveira) [inline screens/src/main/java/com/rjnr/screens/ui/screen/detail_screen/DetailScreen.kt]: Use meaningful names for things

**Comment by jcaiqueoliveira** [comment]
Add a .gitignore file inside your build source folder containing this line: 
/build

**Comment by jcaiqueoliveira** [comment]
> i added a font

Explain the changes that you need, a good practice is to yourself to put comments in the PR explaining the decisions that you took.

File: design/build.gradle.kts [hunk 2]
```diff
@@ -5,7 +5,7 @@ plugins {
 
 android {
     namespace = "com.rjnr.design"
-    compileSdk = ConfigurationData.compileSdk
+    compileSdk = 34
```
Comment (jcaiqueoliveira) [inline design/build.gradle.kts]: the old way its much better, can you revert this change here and in the other modules?

**Review by jcaiqueoliveira — APPROVED** [review]
Approved but pls before merge, fix the comments 🙏

**Comment by RaheemJnr** [comment]
> Approved but pls before merge, fix the comments 🙏

i'll fix the comment and merge
